docs/apis-tools/java-client/job-worker.md

```diff
@@ -5,11 +5,68 @@ description: "Let's take a deeper look at job workers to handle jobs."
 keywords: ["backpressure", "back-pressure", "back pressure"]
 ---
 
+**Job workers are the backbone of process automation in Camunda 8.** They handle automated tasks (service tasks) in your BPMN processes by continuously polling for available jobs and executing your business logic when jobs become available.
+
+This guide covers everything you need to know about implementing and configuring job workers with the Camunda Java Client, from basic concepts to advanced features such as streaming, metrics, and multi-tenancy.
+
+## Quick start
+
+Before diving into the details, here is a simple example of creating a job worker:
+
+```java
+try (final JobWorker workerRegistration = client.newWorker()
+        .jobType(jobType)
+        .handler(new EmailJobHandler())
+        .open()) {
+
+        System.out.println("Job worker opened and receiving jobs of type: " + jobType);
+
+// Keep the worker running
+Thread.sleep(Duration.ofMinutes(10));
+        } catch (InterruptedException e) {
+        throw new RuntimeException(e);
+            }
```

Comment:
🔧 Indentation is a bit off here.

Reply:
Thank you! 
Fixed with the commit 5f9281e854d263d2bdcdb0fbc62a9ca11336408d